aws/data_source_aws_ebs_volume_ids.go

```diff
@@ -0,0 +1,74 @@
+package aws
+
+import (
+	"errors"
+	"log"
+	"strconv"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/terraform-providers/terraform-provider-aws/aws/internal/keyvaluetags"
+)
+
+func dataSourceAwsEbsVolumeIDs() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceAwsEbsVolumeIDsRead,
+		Schema: map[string]*schema.Schema{
+			"filter": ec2CustomFiltersSchema(),
+
+			"tags": tagsSchemaComputed(),
```

Comment:
` tagsSchema()` is better here as `tags` aren't set in the `Read` function.

Reply:
This probably also wants propagating back to `aws_subnet_ids` which uses the same.